spec/steps/report.fmf

```diff
@@ -1,7 +1,51 @@
 summary: Report test results
 
-description:
-    Adjusting notifications about the test progress and results.
+description: |
+    Create report for all the test results. The report is saved into ``report.yaml``
+    file with the following format::
+
+        /plan/one:
```

Comment:
@psss maybe we should have an overall result also across all plans?

Reply:
Yes, makes sense to me to put a `result` key in the top level of the result structure as the overall result.

Reply:
ack

Reply:
fixed, I also put the plans under `plans:` key, mixing together result and tests on top level feels weird ...